tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -226,6 +228,7 @@ internal static void InitializeNoNativeParts()
                 return;
             }
 
+            TracerDebugger.WaitForDebugger(ConfigurationKeys.DebuggerEnabled);
```

Comment:
Isn't the previous `WaitForDebugger` enough?

Reply:
Well there's cases where I test an app with the instrumentation made in AOT mode, so the `Initialize()` method is not called. But `InitializeNoNativeParts()` is called from `CallTargetInvoker`.